### ISSUE TITLE
Fields - Final Accessibility Review

### ISSUE THREAD

**Issue by alschmiedt** [google/blockly#2844]
**Is your feature request related to a problem? Please describe.**
There are some accessibility features that should be implemented for all fields.

**Describe the solution you'd like**
Once we have finished basic field's accessibility we should do a final audit of all fields to see what needs to be added. We should also do some basic testing with screen readers.

To Review:
1. aria-labels
2. Keyboard navigation
3. States
   a. Selected
   b. Expanded 
   c. Active [descendant]
   d. Invalid
4. Semantic/Roles 
5. Ownership of controls
6. Focus/blur
 
**Describe alternatives you've considered**

**Additional context**

**Comment by BeksOmega** [comment]
I'm adding this to triage because I'm not sure what this is actually tracking. Is this about coordinating with an internal partner for testing? If it doesn't represent an actionable task I think it should be closed.

**Comment by rachel-fenichel** [comment]
Closing because this is similar to https://github.com/google/blockly/issues/8205 but less fleshed out.
